src/catalogue/items/catalogueItemDirectoryDialog.component.tsx

```diff
@@ -189,7 +189,7 @@ const CatalogueItemDirectoryDialog = (
               breadcrumbsInfo={catalogueBreadcrumbs}
               onChangeNode={setParentCategoryId}
               onChangeNavigateHome={() => setParentCategoryId(null)}
-              navigateHomeAriaLabel="navigate to catalogue home"
+              homeLocation={'Catalogue'}
```

Comment:
```suggestion
              homeLocation="Catalogue"
```
There could be a few others, just a picky styling comment.